helmfile.d/values/harbor/harbor-backup.yaml.gotmpl

```diff
@@ -7,6 +7,7 @@
 
 dbPassword: {{ .Values.harbor.internal.databasePassword }}
 retentionDays: {{ .Values.harbor.backup.retentionDays }}
+schedule: {{ .Values.harbor.backup.schedule }}
```

Comment:
Consider passing through quote function, to be extra sure it stays a string.

Reply:
Thanks, added now